csi/volumeutils.py

```diff
@@ -1219,7 +1219,7 @@ def yield_pvc_from_mntdir(mntdir):
                 data = json.loads(handle.read().strip())
             logging.debug(
                 logf("Found a PVC at", path=file_path, size=data.get("size")))
-            yield name[name.find("pvc"):name.find(".json")], data.get("size")
+            yield name[name.find("pvc"):name.find(".json")], data.get("size"), , data.get("path_prefix")
```

Comment:
- The extra comma (,) might be a syntax error, pls check
- You changed num of entries in return, earlier we are explicitly using 0 & 1 index, so this isn't a problem. Just mentioning it.

Reply:
Oops, sorry for the typo error